src/views/ModelPlan/CollaborationArea/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useParams } from 'react-router-dom';
-import { Grid, GridContainer } from '@trussworks/react-uswds';
+import { CardGroup, Grid, GridContainer } from '@trussworks/react-uswds';
 // import classNames from 'classnames';
 import {
   //   GetCrtdLsQuery,
```

Comment:
Some commented out stuff here

Reply:
I think you had added these commented code when you were working on the collab area

Reply:
Oops lol yep my b!